recipe/meta.yaml

```diff
@@ -1,42 +1,54 @@
 {% set name = "DoubleMetaphone" %}
 {% set version = "1.1" %}
-{% set md5 = "eb6a3f05f3254db7268d462cd0a3ff49" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  md5: {{ md5 }}
+  # Use upstream archive tarball for tests
+  url: https://github.com/dedupeio/doublemetaphone/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 69eb748a08e8d7d7f4448e8bfbc9b5d5018d979067c9b48c62d121633e4f7b55
 
 build:
-  number: 2
-  script: python setup.py install --single-version-externally-managed --record record.txt
-  skip: true  # [win and py3k]
+  number: 0
```

Comment:
Shouldn't this be bumped to build 3?

Reply:
Unless it is OUR first build.

Reply:
It _is_ our first build.  Unless I can't search 😆